reference-manual/src/main/asciidoc/expert-zone/dag.adoc

```diff
@@ -321,7 +321,7 @@ include::{javasource}/expertzone/Dag.java[tag=s9]
 By default the processor receives items from all inbound edges as they
 arrive. However, there are important cases where an edge must be
 consumed in full to make the processor ready to accept data from other
-edges. A major example is a "`hash join`" which enriches the data stream
+edges. A major example is a `hash-join` which enriches the data stream
```

Comment:
"hash join" isn't a piece of code, it shouldn't use back-ticks. Also shouldn't have the dash: a dash is used when you make one word form multiple, but in this case it's still two words, try to google for hash-join with the dash, i found no example. It could maybe be in italics.

Reply:
I just ran into this while re-reading the refman. As a verb, it should be "hash-join" but as a noun, it should be "hash join".

Reply:
:thinking: there are 22 occurrences of `hash-join`s in the reference manual while only 3 `hash join`s... should I edit all of them?

Reply:
I just saw your comment @mtopolnik, let me go through the usages then

Reply:
Went through all of them, does it make more sense now?